PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/AppCreationScripts/AppCreationScripts.md

```diff
@@ -0,0 +1,146 @@
+# Registering the sample apps with Microsoft Identity Platform and updating the configuration files using PowerShell scripts
+
+## Overview
+
+### Quick summary
+
+1. On Windows run PowerShell and navigate to the root of the cloned directory
```

Comment:
Paths (such as "root of the cloned directory") should be updated now that the sample app isn't in the root